File: src/main/java/net/hectus/neobb/shop/RandomizedShop.java
package net.hectus.neobb.shop;

import net.hectus.neobb.player.NeoPlayer;
import org.jetbrains.annotations.NotNull;

public class RandomizedShop extends Shop {
    public RandomizedShop(@NotNull NeoPlayer player) {
        super(player);
    }

    @Override
    public void open() {
        player.inventory.fillInRandomly();
        done();
    }
}

File: src/main/kotlin/net/hectus/neobb/shop/RandomizedShop.kt
package net.hectus.neobb.shop

import net.hectus.neobb.player.NeoPlayer
import net.hectus.neobb.util.bukkitRunLater

class RandomizedShop(player: NeoPlayer) : Shop(player) {
    override fun open() {
        bukkitRunLater(10) {
            player.inventory.fillInRandomly()
            done()
        }
    }
}
